dags/bqetl_nondesktop.py

```diff
@@ -56,42 +56,38 @@
         dag=dag,
     )
 
-    wait_for_telemetry_derived__core_clients_last_seen__v1 = ExternalTaskSensor(
-        task_id="wait_for_telemetry_derived__core_clients_last_seen__v1",
-        external_dag_id="bqetl_core",
-        external_task_id="telemetry_derived__core_clients_last_seen__v1",
-        execution_delta=datetime.timedelta(seconds=3600),
+    wait_for_baseline_clients_last_seen = ExternalTaskSensor(
+        task_id="wait_for_baseline_clients_last_seen",
+        external_dag_id="copy_deduplicate",
+        external_task_id="baseline_clients_last_seen",
+        execution_delta=datetime.timedelta(seconds=7200),
         check_existence=True,
         mode="reschedule",
     )
 
     telemetry_derived__firefox_nondesktop_day_2_7_activation__v1.set_upstream(
-        wait_for_telemetry_derived__core_clients_last_seen__v1
+        wait_for_baseline_clients_last_seen
     )
-    wait_for_copy_deduplicate_baseline_clients_last_seen = ExternalTaskSensor(
-        task_id="wait_for_copy_deduplicate_baseline_clients_last_seen",
-        external_dag_id="copy_deduplicate",
-        external_task_id="baseline_clients_last_seen",
-        execution_delta=datetime.timedelta(seconds=7200),
+    wait_for_telemetry_derived__core_clients_last_seen__v1 = ExternalTaskSensor(
+        task_id="wait_for_telemetry_derived__core_clients_last_seen__v1",
+        external_dag_id="bqetl_core",
+        external_task_id="telemetry_derived__core_clients_last_seen__v1",
+        execution_delta=datetime.timedelta(seconds=3600),
         check_existence=True,
         mode="reschedule",
-        dag=dag,
     )
 
     telemetry_derived__firefox_nondesktop_day_2_7_activation__v1.set_upstream(
-        wait_for_copy_deduplicate_baseline_clients_last_seen
+        wait_for_telemetry_derived__core_clients_last_seen__v1
     )
 
     telemetry_derived__firefox_nondesktop_exact_mau28__v1.set_upstream(
-        wait_for_telemetry_derived__core_clients_last_seen__v1
+        wait_for_baseline_clients_last_seen
     )
     telemetry_derived__firefox_nondesktop_exact_mau28__v1.set_upstream(
-        wait_for_copy_deduplicate_baseline_clients_last_seen
+        wait_for_telemetry_derived__core_clients_last_seen__v1
     )
 
     firefox_nondesktop_exact_mau28_by_client_count_dimensions.set_upstream(
         wait_for_telemetry_derived__core_clients_last_seen__v1
     )
-    firefox_nondesktop_exact_mau28_by_client_count_dimensions.set_upstream(
-        wait_for_copy_deduplicate_baseline_clients_last_seen
-    )
```

Comment:
This dependency would get removed. I don't think it is actually required, but if it is then it can be added back using `depends_on`

Reply:
Indeed, looks like it hits only core pings. This is probably an oversight. I will file an issue about it. But for this PR, removing this dependency looks correct.

Reply:
https://github.com/mozilla/bigquery-etl/issues/1184